Podcast/SettingsPageViewController.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  SettingsPageViewController.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 10/22/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+// Add more field types as created
+// String value is cell reuse ID
+enum SettingsFieldType {
+    case button
+    case textField
+    case disclosure
+    case label
+    case toggle
+    
+    static let allValues: [SettingsFieldType] = [.button, .textField, .disclosure, .label, .toggle]
+    
+    var reuseIdAndClass: (String, AnyClass) {
```

Comment:
let's put the reuseIdentifier on the tableViewCell as a field so then we can just have a tableViewCell variable on this enum 
I think the tuple return with this unspecific field name might get confusing later on

Reply:
Yeah that sounds good!

Reply:
Actually can't do that, classes can't override static variables, and I don't want to instantiate an entire class just to get a string value out.